{"name": "Тест невозможности авторизации зарегистрированного курьера при вводе некорректных данных (статус ответа 404)", "status": "passed", "steps": [{"name": "Авторизация курьера", "status": "passed", "parameters": [{"name": "login", "value": "'hntbkzqxcw'"}, {"name": "password", "value": "'incorrect_data'"}], "start": 1734893591831, "stop": 1734893592020}], "parameters": [{"name": "field_name", "value": "'password'"}], "start": 1734893591830, "stop": 1734893592021, "uuid": "c3e08c36-bee0-4b5f-aa21-560cc8858c14", "historyId": "78bd892d92288947a085d1996bcd5f2b", "testCaseId": "62de0a1455e4348f76fb13991ea19ed5", "fullName": "tests.couriers.test_login_courier.TestLoginCourier#test_login_courier_with_incorrect_login_or_password", "labels": [{"name": "parentSuite", "value": "tests.couriers"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-TPCCV60"}, {"name": "thread", "value": "12028-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.couriers.test_login_courier"}]}